package.json

```diff
@@ -23,10 +23,11 @@
     "retry": "0.6.1",
     "sdp-interop": "0.1.11",
     "sdp-simulcast": "0.1.11",
-    "sdp-transform": "1.5.3",
+    "sdp-transform": "^2.0.3",
```

Comment:
I don't think any of these changes are necessary at this stage. When you do introduce them, please use exact versions.